lms/templates/course_modes/choose.html

```diff
@@ -163,11 +441,50 @@ <h4 class="title">${_("Audit This Course")}</h4>
                                 </ul>
                             </div>
                         % elif "audit" in modes:
-                            <span class="deco-divider">
+                            <span class="deco-divider v1">
+                                <span class="copy">${_("or")}</span>
+                            </span>
+                            <!-- This div was added as part of the LEARNER-1312 test. The v2 class should be removed if this test is implemented-->
+                            <span class="deco-divider v2 hidden">
                                 <span class="copy">${_("or")}</span>
                             </span>
 
-                            <div class="register-choice register-choice-audit">
+                            <!-- This div was added as part of the LEARNER-1312 test. The v2 class should be removed if this test is implemented-->
+                            <div class="register-choice register-choice-donate v2 hidden">
+                                <h4 class="title">
+                                    Donate to Support our Non-Profit Mission
+                                </h4>
+                                <div class="wrapper-copy-inline">
+                                    <div class="wrapper-copy">
+                                        Any amount will support our mission to make the world's best education more accessible.
+                                    </div>
+                                    <img src="/static/images/edx-home-graphic.png" class="visual-reference img-donate" alt="Visual of two hands forming a heart shape" >
+                                </div>
+                                <div class="copy-inline">
+                                    <ul class="list-actions">
+                                      <li class="action action-select">
+                                        <a class="donation-link" href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&amp;hosted_button_id=MJUBGL54QTCTA">Donate and Continue to Course</a>
+                                      </li>
+                                    </ul>
+                                </div>
+                            </div>
+
+                            <!-- This div was added as part of the LEARNER-1312 test. The v2 class should be removed if this test is implemented-->
```

Comment:
unless you feel very strongly i'd rather leave it like this.  
I think it will make cleaning up the test super simple as me or whoever does that ticket will just have to search LEARNER-1312 and remove all those spots